src/get-started/test-service.md

```diff
@@ -322,19 +320,23 @@ with a specified public key:
 fn assert_no_wallet(&self, pub_key: PublicKey) {
     let err = self
         .inner
-        .public(ApiKind::Service("cryptocurrency"))
+        .public(ApiKind::Service(INSTANCE_NAME))
         .query(&WalletQuery { pub_key })
         .get::<Wallet>("v1/wallet")
         .unwrap_err();
 
-    assert_matches!(err, Error::NotFound(ref body) if body == "Wallet not found");
+    assert_eq!(err.http_code, api::HttpStatusCode::NOT_FOUND);
```

Comment:
It could be worth explaining here how errors returned by Rust services are structured.